Task-1/fs-demo.js

```diff
@@ -0,0 +1,45 @@
+const fs = require("fs");
+const crypto = require("crypto");
+const argv = process.argv.slice(2);
+const input = argv[0];
+fs.writeFileSync("output.txt", "--- Generating Fresh Output file ----\n");
+if (input != null) {
+  const stat = fs.statSync(input);
+  if (stat.isDirectory()) {
+    readdirectory(input);
+  } else {
+    console.log("Not a directory");
+  }
+} else {
+  console.log("No input");
+}
+
+function readdirectory(dirName) {
+  const dir = fs.readdirSync(dirName);
+  dir.forEach(async function(dirent) {
+    const path = dirName + "/" + dirent;
+    if (fs.statSync(path).isDirectory()) {
+      readdirectory(path);
+    } else {
+      let output =
+        path +
+        "\t-[MD5]-> " +
+        calculateHash(path, "md5") +
+        " -[SHA1]-> " +
+        calculateHash(path, "sha1") +
+        "\n";
+      writtingoutputfile(output);
```

Comment:
use camel casing.. for better readability. also for the parent function `readdirectory`.

also you will have to await on this as you are using FileSync in the function.

Reply:
changes are done